### PR TITLE
Remove examples moved to docs site

### PR DESCRIPTION
**Pull request by erictune**
Part of https://github.com/kubernetes/kubernetes/issues/23511
Left redirect in md files and deleted files intended for download.

---

This change is [Reviewable](http://reviewable.k8s.io/reviews/kubernetes/kubernetes/23513)

### CODE REVIEW

**Comment by bgrant0607** [comment]
Thanks. We need to get examples tests working in the docs repo.
cc @kubernetes/docs @kubernetes/examples

File: examples/job/expansions/README.md [hunk 1]
```diff
@@ -21,7 +22,7 @@ refer to the docs that go with that version.
 <!-- TAG RELEASE_LINK, added by the munger automatically -->
 <strong>
 The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.2/examples/job/expansions/README.md).
+[here](http://releases.k8s.io/release-1.2/docs/user-guide/jobs.md).
```
Comment (bgrant0607) [inline examples/job/expansions/README.md]: I suggest http://kubernetes.io/docs/user-guide/jobs/

File: examples/job/expansions/README.md [hunk 2]
```diff
@@ -32,218 +33,9 @@ Documentation for other releases can be found at
...
-  concurrently.  One reason would be to avoid using too many
-  compute resources.  Another would be to limit the number of
-  concurrent requests to a shared resource, such as a database,
-  used by all the pods in the job.
-- very large numbers of jobs created at once overload the
-  kubernetes apiserver, controller, or scheduler.
-
-In this case, you can consider one of the
-other [job patterns](../../../docs/user-guide/jobs.md#job-patterns).
+This file has moved to: http://kubernetes.github.io/docs/user-guide/job/examples
```
Comment (bgrant0607) [inline examples/job/expansions/README.md]: I get 404.

Also, kubernetes.io is preferred

File: examples/job/expansions/README.md [hunk 3]
```diff
@@ -32,218 +33,9 @@ Documentation for other releases can be found at
...
 
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/examples/job/expansions/README.md?pixel)]()
+[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/docs/user-guide/jobs.md?pixel)]()
```
Comment (bgrant0607) [inline examples/job/expansions/README.md]: I'm not sure it makes sense to change the analytics URL.

**Comment by erictune** [comment]
All comments addressed

**Comment by zmerlynn** [comment]
This was sitting in my PR queue but I didn't notice it because I subtracted `needs-rebase` for a while. Do you want to get it in soon?

**Comment by erictune** [comment]
Rebased, PTAL

**Comment by zmerlynn** [comment]
I'm not clear why this isn't at the top of the SQ again (it was a bit ago), but trying to ensure it slips in before another rebase.

**Pull request merged**